DeepSpeech.py

```diff
@@ -1042,11 +1044,13 @@ def train():
     # Possibly restore checkpoint
     start_epoch = 0
     if restore_checkpoint:
+        # Restore checkpoint created after epoch N, restart from epoch N+1
         checkpoint = tf.train.get_checkpoint_state(checkpoint_dir)
         if checkpoint and checkpoint.model_checkpoint_path:
             hibernation_path = checkpoint.model_checkpoint_path
-            start_epoch = int(checkpoint.model_checkpoint_path.split('-')[-1])
-            print 'Resuming training from epoch %d' % (start_epoch + 1)
+            last_epoch = int(checkpoint.model_checkpoint_path.split('-')[-1])
+            start_epoch = last_epoch+1
```

Comment:
Yeah. However, the 
```python
train_wer = 0.0
dev_wer = 0.0
```
results will get published to [doyouunderstand.me ](http://doyouunderstand.me/) which is not what we want in this situation.

But, it seems like a real fix, save the values externally, would then require changes in the web publishing code too.

Or one could decouple training from WER calculation allowing one to calculate training WER with out training for example. But again that's real work not a hot fix.

I'm not sure the right way forward, but I think having  [doyouunderstand.me ](http://doyouunderstand.me/) display WER 0.0 is not something we should do.